html/global_attributes.json

```
@@ -1143,6 +1143,83 @@
           }
         }
       },
+      "inert": {
+        "__compat": {
+          "mdn_url": "https://developer.mozilla.org/docs/Web/HTML/Global_attributes/inert",
+          "spec_url": "https://html.spec.whatwg.org/multipage/interaction.html#the-inert-attribute",
+          "support": {
+            "chrome": {
+              "version_added": "102"
+            },
+            "chrome_android": {
+              "version_added": "102"
+            },
+            "edge": {
+              "version_added": "102"
+            },
+            "firefox": {
+              "version_added": "81",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "html5.inert.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            },
+            "firefox_android": {
+              "version_added": "81",
+              "flags": [
+                {
+                  "type": "preference",
+                  "name": "html5.inert.enabled",
+                  "value_to_set": "true"
+                }
+              ]
+            },
```

Firefox Android actually doesn't have flag support anymore!

```suggestion
            "firefox_android": {
              "version_added": false
            },
```